Bottom: cab8634891f10a65b66f99ad82009e6b67afb98f
Top:    b2421756731620d61324cc3e558ff8e55fafc478
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-01 16:55:07 +0000

Tmp debug


---

diff --git a/src/autoroute.c b/src/autoroute.c
index 58ff351..2b88194 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -79,6 +79,7 @@
 #include "undo.h"
 #include "vector.h"
 #include "pcb-printf.h"
+#include "hid_draw.h"
 
 #ifdef HAVE_LIBDMALLOC
 #include <dmalloc.h>
